.github/buildomat/jobs/check-features.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+#:
+#: name = "check-features (helios)"
+#: variety = "basic"
+#: target = "helios-2.0"
+#: rust_toolchain = true
+#: output_rules = []
+
+# Run cargo check on illumos with feature-specifics like `no-default-features`.
+
+set -o errexit
+set -o pipefail
+set -o xtrace
+
+cargo --version
+rustc --version
+
+#
+# Set up our PATH for use with this workspace.
+#
+source ./env.sh
+
+banner prerequisites
+ptime -m bash ./tools/install_builder_prerequisites.sh -y
+
+banner check
+export CARGO_INCREMENTAL=0
+ptime -m cargo check --workspace --bins --tests --no-default-features
+RUSTDOCFLAGS="--document-private-items -D warnings" ptime -m cargo doc --workspace --no-deps --no-default-features
+
+#
+# `cargo-hack` check feature-powerset
+#
+banner hack
+cargo install cargo-hack --locked
```

Comment:
I think we should use a pre-built binary here if possible, and pin the version of the tool.

Reply:
So, I went away from the pre-built binary, particularly since it's an external tool. I pushed up the xtask which will install it locally to a specific version (or default floats on patch), but also allows for installing it separately from the xtask. See what you think.